### ISSUE TITLE
[question] Having Modal open on page load in Nuxt/Vue

### ISSUE THREAD

**Issue by gkurl** [CodyHouse/cd-modal-window-changelog#7]
Hi Team,

I would appreciate your assistance on this - I am trying to get a modal window to open up as the first thing the user sees when the site is access from any page.

I have contained all the HTML, SCSS and JS for the Modal, in a separate `AgeCheckModal` component.

I am then including this component in my `default.vue` layout which is applicable to all pages - and then I am attempting to follow the information you have provided on the info page to open the modal without having a trigger i.e. button. I am calling the `openModal` method in my `mounted` hook. 

No matter what I've tried - the modal just renders statically and doesn't overlay the whole site like in the demo - this is what I currently see - it's just like any other HTML that is a part of the page - I'm not using button as I need it on page load.

<img width="1406" alt="image" src="https://github.com/codyhouse/cd-modal-window-changelog/assets/34505859/36bf451d-d29b-468c-bc65-4dae3c191648">

I am basically going to use it as an age verification pop-up. So hence I need to present this on page/website load.

I have tried creating a modal element on it's own, on a new separate page that isn't `index.vue` or `default.vue`, and it works with the button press so the JS etc is definitely there and loading.

So basically how to hide from view and open on `this.openModal` call and perform some logic after.

Any help would be massively appreciated.

**Comment by claudia-romano** [comment]
Hi there, from your screenshot it looks like the CSS of the modal is not properly applied (the modal should cover the entire page). I suggest you double check that the modal classes have been imported properly (e.g., the `.modal` element should have a fixed position and height and width to 100%).

**Comment by gkurl** [comment]
Sorry for the delayed reply - I'm still looking into it, as far as I know I have the CSS setup correctly within the Modal component. I'll double check everything and show you screenshots etc.
